### PR TITLE
Dockerize

### PR DESCRIPTION
**Pull request by BenMukebo**
- [ ]  Create a Dockerfile in the root directory of your Rails application.
- [ ] Build the Docker image.
- [How to Containerize and Deploy Apps With Docker and Heroku](https://betterprogramming.pub/how-to-containerize-and-deploy-apps-with-docker-and-heroku-b1c49e5bc070)
- [Dockerizing a Ruby on Rails Application](https://semaphoreci.com/community/tutorials/dockerizing-a-ruby-on-rails-application)

### CODE REVIEW

**Comment by BenMukebo** [comment]
Hi @vic778, I'm really stuck on this issue, i need help!!

**Comment by BenMukebo** [comment]
@vic778 I’m around, we can pair!!

**Comment by vic778** [comment]
join 
https://meet.google.com/mag-xjmu-mxf

**Comment by Tresor11** [comment]
@BenMukebo, The scope of this PR, is not clear. Why are there so many changes in the role and user models?

**Review by Tresor11 — CHANGES_REQUESTED** [review]

File: db/migrate/20230415132731_add_data_info_address_to_users.rb [hunk 1]
```diff
@@ -0,0 +1,24 @@
+class AddDataInfoAddressToUsers < ActiveRecord::Migration[7.0]
```
[the remark this answers is not in the document]
Comment (BenMukebo) [reply db/migrate/20230415132731_add_data_info_address_to_users.rb:1]: @Tresor11, I open this PR while I was waiting for you to approve the previous PR that is why you're seeing even the changes from the previous PR !!

**Pull request closed without merging**